src/conference/lyonJS/Replay.tsx

```diff
@@ -27,15 +27,20 @@ export const ReplayLyonJS: React.FC<LyonJSReplayType> = ({
 	date,
 	sponsor,
 }) => {
-	console.log(date);
 	return (
 		<AbsoluteFill style={{backgroundColor: 'white', overflow: 'hidden'}}>
```

Comment:
We should definitely add this template to the website to @CruuzAzul :)

Reply:
#185 👋